validationFunction/README.md

```diff
@@ -0,0 +1,27 @@
+# Aim/Objective
+
+The purpose of the Ottawa Data Model (ODM) is to create an open science structure which can be used as a guideline for future models which promote data sharing.
```

Comment:
Wrong name for the ODM.

Ideally, there would be a sngle H1 header with a more descriptive title (what is this document about)? 
This describers the .sv validsaiton file, but I've seen yaml and json versions in the PR too. Shouldn't these be described as well?
What about the sample script?